camel/societies/workforce/workforce.py

```
@@ -46,6 +46,35 @@
 
 logger = get_logger(__name__)
 
+# define timeout threshold
+TIMEOUT_THRESHOLD = 180.0  # Default timeout in seconds
+
+# define type variable for with_timeout function
+T = TypeVar("T")
+
+async def with_timeout(
+    coro: Awaitable[T],
```

Ruff (0.11.9)

Undefined name `Awaitable`

(F821)